### DISCUSSION TITLE
Using this component within MudBlazor makes the Insert Table icon render weird.

### DISCUSSION THREAD

**Discussion by 007darkmatter5** [erossini/BlazorMarkdownEditor#34] [Q&A] [answered]
When I use this component withing MudBlazor the Insert Table icon in the toolbar jumps down a row and fills the width of the component.  See the image I've included.  Any suggestions on how to fix it?

![Markdown](https://github.com/erossini/BlazorMarkdownEditor/assets/2148432/6471cb6d-a6d1-4bce-a61a-af22779ffa0a)

**Comment by erossini** [comment]
Yes, I noticed that. This is because the name of the button is `table`. Today, I'll release a new version where the names are changed to avoid this issue.

**Reply by 007darkmatter5** [reply]
Do I just need to update the package through NuGet Package Manager once the update is pushed?

**Reply by erossini** [reply]
> Do I just need to update the package through NuGet Package Manager once the update is pushed?

Yes, I have just published the new version.

**Reply by 007darkmatter5** [reply]
I pulled the update, My project shows.

`<PackageReference Include="PSC.Blazor.Components.MarkdownEditor" Version="8.0.2" />`

but it's still showing the same problem.  Did I miss something?

**Reply by 007darkmatter5** [reply, marked as answer]
It started working after a few reloads.  Thank you for your work on this!!

**Comment by erossini** [comment]
Thanks for confirming.
Enrico
